kpops/cli/main.py

```diff
@@ -185,7 +190,7 @@ def create_pipeline_config(
     return pipeline_config
 
 
-@app.command(
+@app.command(  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
opened an issue https://github.com/rec/dtyper/issues/8

Reply:
I think it would be better to put the link on the code too :)